src/Hyperwallet/Model/TransferMethod.php

```diff
@@ -1121,4 +1122,23 @@ public function setPostalCode($postalCode) {
         return $this;
     }
 
+    /**
+     * Get the is default transfer method
+     *
+     * @return string
+     */
+    public function getIsDefaultTransferMethod() {
+        return $this->isDefaultTransferMethod;
+    }
+
+    /**
+     * Set the is default transfer method
+     *
+     * @param string $isDefaultTransferMethod
+     * @return PayPalAccount
```

Comment:
@return TransferMethod will be more appropriate. Please check and update

Reply:
Changes addressed